src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -312,6 +321,78 @@ impl ScalarFunc {
             _ => Err(box_err!("invalid direction value: {}", direction)),
         }
     }
+
+    #[inline]
+    pub fn to_base64<'a, 'b: 'a>(
+        &'b self,
+        ctx: &mut EvalContext,
+        row: &'a [Datum],
+    ) -> Result<Option<Cow<'a, [u8]>>> {
+        let s = try_opt!(self.children[0].eval_string(ctx, row));
+        if encoded_size(s.len()).is_none() {
+            return Ok(Some(Cow::Borrowed(b"")));
+        }
+        // TODO:
+        // 1. check with maxAllowedPacket
```

Comment:
oh I see,  mysql.MaxBlobWidth is const, thanks.
how about  `MaxAllowedPacket` ?   is has an initial value  `67108864`, but can be changed.
in TiDB, it can be fetched by `ctx.GetSessionVars().GetSystemVar(variable.MaxAllowedPacket)`.
how can I get this in TiKV?

Reply:
@breeswish Do you have any suggestion?